.github/workflows/ci.yaml

```diff
@@ -155,6 +151,48 @@ jobs:
           docker build -f docker/pmbgpd/Dockerfile -t pmbgpd:_build .
```

Comment:
maybe something you dont want to cover in this PR, but I'd recommend you to try docker buildkit as an image builder, with caching enabled, as described [here](https://evilmartians.com/chronicles/build-images-on-github-actions-with-docker-layer-caching). That will improve performance not just for this stage, but also for the previous ones

Reply:
Thanks for the pointer. Definitely out of the scope of this PR.

As the article mentions, you need to use an external docker image repository and push it, to get the advantage. I will have a look, as for the builders, specially, that could speed up quite a bit the builds.

For the pmacct containers, I am not sure. Probably the common part, but all these derived containers just change the entry point so building is pretty cheap.